.gitignore

```diff
@@ -7,3 +7,5 @@ target/
 tags
 
 .bsp
+
+.idea
```

Comment:
Can you revert this change? You shouldn't be adding IDE specific files to a projects gitignore (instead users should be using a global gitignore on their machine, see https://stackoverflow.com/a/7335487

Reply:
> Can you revert this change? You shouldn't be adding IDE specific files to a projects gitignore (instead users should be using a global gitignore on their machine, see https://stackoverflow.com/a/7335487

fixed.

Reply:
@mdedetrich ^